src/utils/apiCalls.js

```diff
@@ -52,3 +52,68 @@ export const addPalette = async (newPalette) => {
     return addedPalette;
 };
 
+export const deletePalette = async (paletteId) => {
+  const options = {
+    method: 'DELETE',
+    headers: {
+      'content-type': 'application/json'
+    }
+  };
+  const response = await fetch(`${baseUrl}/palettes/${paletteId}`, options);
+  if (!response.ok) {
+    throw Error('Unable to delete the palette.')
+  };
+  const message = await response.json();
+  return message;
```

Comment:
I looked up appropriate responses for a delete request, and it varied, but it seemed like returning the deleted ID was standard. I did see your comment for the endpoints, and we can change it to include a message, but I figured we should get together and decide on a path forward.

Reply:
Good thought, we can definitely chat more about this when we meet tomorrow.